src/edge/simulator.ts

```diff
@@ -281,9 +282,7 @@ export class Simulator {
             req.on("response",  (res) => {
                 if (res.statusCode === 200) {
                     req.pipe(unzipper.Extract({ path: Simulator.WindowsStandaloneSimulatorFolder }))
-                    .on("close", () => resolve()).on("error", (e) => reject(e));
-                } else if (res.statusCode === 404) {
```

Comment:
404 is not specially handled?

Reply:
Because use cannot set the version or download url manually, , so we do not need to special handle the 404 case